esp/public/media/scripts/onsite/ajax_status.js

```diff
@@ -17,7 +17,9 @@ var settings = {
     override_full: false,
     disable_grade_filter: false,
     compact_classes: true,
-    categories_to_display: []
+    categories_to_display: [],
+    selected_categories: [],
+    is_initial: true,
```

Comment:
I'm a bit rusty with Javascript syntax; is this trailing comma going to be a problem for any browsers that people actually use?  (Wasn't a problem for me.)